src/components/ContributionsEpicButtons.test.tsx

```diff
@@ -5,41 +5,59 @@
 import '@testing-library/jest-dom';
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import { factories } from '../factories';
+import testData from './ContributionsEpic.testData';
 
-import { EpicButtons } from './EpicButtons';
+import { ContributionsEpicButtons } from './ContributionsEpicButtons';
 
 describe('EpicButtons', () => {
     it('Renders the primary button', () => {
-        const variant = factories.variant.build({
+        const variant = {
+            ...testData.content,
+            name: 'Example',
+            showTicker: false,
             cta: {
                 text: 'Button text!',
+                baseUrl: 'https://support.theguardian.com/support',
             },
-        });
-        const tracking = factories.tracking.build();
+        };
+        const tracking = testData.tracking;
 
-        render(<EpicButtons variant={variant} tracking={tracking} />);
+        render(
+            <ContributionsEpicButtons
+                variant={variant}
+                tracking={tracking}
+                onOpenReminderClick={() => console.log('Reminder was clicked')}
```

Comment:
I think this is necessary just to satisfy the interface to the component (we're not expecting it to be called here)? In cases like this I like I like to define a named no-op function and pass that to make the intent super clear. What do you reckon?

```js
const noOp =() => {};
render(
            <ContributionsEpicButtons
                variant={variant}
                tracking={tracking}
                onOpenReminderClick={noOp}
            />
)